Index: DBproject/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/DBproject/__init__.py b/DBproject/__init__.py
--- a/DBproject/__init__.py	(revision e764f334d527a7bf5ee06280991bc778103cb207)
+++ b/DBproject/__init__.py	(date 1748789543619)
@@ -1,0 +1,1 @@
+# test
\ No newline at end of file
